### ISSUE TITLE
Generate Code Snippets does not encode HTTP request line correctly

### ISSUE THREAD

**Issue by nlwillia** [a85/POSTMan-Chrome-Extension#2780]
Postman for Chrome 4.10.3 on Windows 10

If you click the Code link to preview your request or generate a code snippet, the HTTP format does not encode the query parameters in the URI of the request line correctly.

````
GET /someresource?a=1&amp;b=2 HTTP/1.1
````
It should be:
````
GET /someresource?a=1&b=2 HTTP/1.1
````
None of the other formats have the problem.

**Comment by sdnts** [comment]
Hi @nlwillia Thanks for letting us know, we'll look into this.

**Comment by abhijitkane** [comment]
@nlwillia 
Can you try this is Postman 7.2.1, with the 'Experimental Codegen Mode' flag enabled in Postman's global settings? We've introduced bug-fixes, support for two new languages (JS-Fetch and Powershell), and the ability to configure certain aspects of the generated snippet.

**Comment by nlwillia** [comment]
Confirmed that the issue seems to be fixed by the flag in 7.2.1.  With the flag off, the original problem still occurs.  With the flag on, it encodes correctly.

**Comment by abhijitkane** [comment]
@nlwillia Yep - we'll soon be making that the default. Thanks for confirming!
